packages/api/src/router/post.ts

```diff
@@ -13,13 +13,15 @@ export const postRouter = createTRPCRouter({
       z.object({
         title: z.string().trim().min(1).max(300),
         contents: z.string().min(1),
+        postType: z.enum(["Announcement", "Discussion"]),
       }),
     )
     .mutation(async ({ ctx, input }) => {
       await ctx.db.post.create({
         data: {
           title: input.title,
           contents: input.contents,
+          postType: input.postType,
```

Comment:
can u not add this to the routes -- going to separate the routes for announcements and discussions (announcements will be admin only)

Reply:
for now you can just use discussion as the default

Reply:
Made separate endpoints for discussions and announcements in d10b7b5. Let me know if that's what you meant